doc/util/gen-package-list.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+import base64
+import requests
+from pprint import pprint
+import re
+
+# get authentication token for GitHub
+secret = open("secret-token.txt", "r+")
+info = secret.readline().split(",")
+user = info[0].strip()
+token = info[1].strip()
+secret.close()
+
+f = open("../rst/tools/mason/package-list.rst", "w+")
```

Comment:
Will running this script more than once without some sort of 'clean' step cause issues?

Reply:
No, the current one will be overwritten